src/utils/authentication/utils/getUrlParameter.js

```diff
@@ -0,0 +1,8 @@
+export default (key) => {
+  const cleanKey = key.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]')
+  const regex = new RegExp(`[\\?&]${cleanKey}=([^&#]*)`)
```

Comment:
I personally feel the file name is good enough: you are getting a url parameter from a key. So if the url was `https://website.com?answer=42` and you ran `getUrlParameter('answer')` you would get 42. I could make a test for it if that would help clarify though.